### DISCUSSION TITLE
Captions should be enabled by default for images within the posts section.

### DISCUSSION THREAD

**Discussion by bershatsky** [adityatelange/hugo-PaperMod#601] [Ideas]
I should be able to use markdown like this (ala iA writer) to generate a caption below an image.

`My ![alt attribute goes here!](/path/to/train.jpg "This is a Title" )`

Is there any way to make this work?

**Comment by kdkasad** [comment]
Use Hugo's `figure` short code instead. It's supported by PaperMod whereas standard markdown images aren't (AFAIK).
